docs/docs/migrating-from-v1-to-v2.md

```diff
@@ -394,3 +395,10 @@ Import graphql types from `gatsby/graphql` to prevent `Schema must contain uniqu
 -const { GraphQLString } = require(`graphql`)
 +const { GraphQLString } = require(`gatsby/graphql`)
 ```
+
+## Move Babel Configuration
+The latest version of Gatsby uses Babel 7, which introduced [a new behavior for configuration lookup / resolution](https://github.com/babel/babel/issues/6766).  In the case where a _.babelrc_ file might have been used at the root of the project, like for configuring Jest, moving that Babel configuration into _jest.config.json_ will avoid any conflicts.
+
+[This GitHub comment](https://github.com/facebook/jest/issues/1468#issuecomment-361260279) documents the steps needed to do that.
+
+More information on Gatsby and Babel configuration available [here](https://v2--gatsbyjs.netlify.com/docs/babel/#how-to-use-a-custom-babelrc-file). 
```

Comment:
@m-allanson I think we should avoid using temporary link `https://v2--gatsbyjs.netlify.com/` in any place, I just closed a similar PR #5870